CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - feat: Add transaction to the scope #992
 - fix: Pass SentryTracer to span child #1040
 - feat: Add span to SentrySDK #1042
+- feat: Add urlSessionDelegate option to SentryOptions #965
```

Comment:
```suggestion
- feat: Add urlSessionDelegate option to SentryOptions #1044
```

You can link the issue in this PR.
CHANGELOG entries need to link to the PR where things were changed/added.